### PR TITLE
Rh_kernel_update: enlarge timeout of install_rpm to 30mins to avoid case failed in bad network traffic.

### PR DESCRIPTION
**Pull request by quanwenli**
Maxinum 15 mins spend in my 5 tests, set double redundancy for extreme network.

Signed-off-by: Wenli Quan <wquan@redhat.com>

id : 1469028

### CODE REVIEW

**Comment by quanwenli** [comment]
(1/1) smp_16.4096m.repeat1.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rh_kernel_update: PASS (927.43 s)

File: qemu/tests/rh_kernel_update.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def get_brew_url(mnt_path, download_root):
         logging.debug("Brew URL is %s" % url)
         return url
 
-    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=600):
+    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=3600):
```
Comment (CongLi) [inline qemu/tests/rh_kernel_update.py]: it's not a good idea to update the default value, better to update as a configurable param.

Comment (CongLi) [inline qemu/tests/rh_kernel_update.py]: IMO, the default value = 3600 will be overwritten if there is another value passed.
But @luckyh  thinks the caller should ensure that, then it's acceptable.

Comment (CongLi) [inline qemu/tests/rh_kernel_update.py]: It's ok for me but IMO it's not the best solution.

**Pull request merged**